src/daemon/environment.c

```diff
@@ -33,6 +34,37 @@ static const char *verify_or_create_required_private_directory(const char *dir)
     return verify_required_directory(dir);
 }
 
+const char *detect_libcurl_default_ca() {
+    curl_version_info_data *info = curl_version_info(CURLVERSION_NOW);
+    if (info) {
+
+        // Check built-in CA bundle
+        if (info->cainfo && access(info->cainfo, R_OK) == 0)
+            return info->cainfo;
+    }
+
+    return NULL;
+}
+
+static const char *detect_ca_path(void) {
+    static const char *paths[] = {
+        "/etc/ssl/certs/ca-certificates.crt",               // Debian, Ubuntu, Arch
+        "/etc/pki/tls/certs/ca-bundle.crt",                 // RHEL, CentOS, Fedora
+        "/etc/ssl/ca-bundle.pem",                           // OpenSUSE
+        "/etc/ssl/cert.pem",                                // Alpine
+        "/opt/netdata/etc/ssl/certs/ca-certificates.crt",   // Netdata static build
+        "/opt/netdata/share/ssl/certs/ca-certificates.crt", // Netdata static build - fallback
```

Comment:
This should, again for consistency with the TLS implementation, not be on the list. If the installer determines we need it, then it will set up appropriate symlinks so that it is used, but that’s the only case when we should be using it.